misc/main.cpp

```diff
@@ -96,6 +96,11 @@ static void parseArguments(QString *fileName, int argc, char* argv[])
 
 int main(int argc, char* argv[])
 {
+  // must be called before QApplication is created
+#if QT_VERSION >= QT_VERSION_CHECK(5,6,0)
+  QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
```

Comment:
I see what you mean. Next time. I use tabs. Otherwise a .editorconfig comes in handy to make sure the users editors is set up correctly: https://github.com/NixOS/nixpkgs/blob/master/.editorconfig